pkg/datapath/loader/base.go

```diff
@@ -307,9 +310,13 @@ func (l *loader) reinitializeXDPLocked(ctx context.Context, extraCArgs []string)
 // and reinsertion of the object into the kernel as well as an atomic program replacement
 // at the XDP hook. extraCArgs can be passed-in in order to alter BPF code defines.
 func (l *loader) ReinitializeXDP(ctx context.Context, o datapath.BaseProgramOwner, extraCArgs []string) error {
+	// TODO: react to changes (using the currently ignored watch channel)
```

Comment:
Not a big fan of TODOs. Something that needs to happen in this PR? Something that needs a follow up issue?

Reply:
Fair enough. It won't happen in this PR and it's tracked in an issue in a different repo - do you prefer not having the TODO in that case?

Reply:
Yes, but that is just a nit.